### DISCUSSION TITLE
Deleted Items

### DISCUSSION THREAD

**Discussion by hitguba248** [lncrawl/lightnovel-crawler#1691] [Q&A] [answered]
Hi, I'm new to this tool and I may have just shot myself in the foot and deleted a directory after misspelling and choosing the 'delete and start fresh' option. when downloading a new novel.  I don't see the files in question in my recycle bin though, is everything permanently gone or did lncrawl move things to a hidden directory.  Is there any chance of recovery here?

**Comment by dipu-bd** [comment, marked as answer]
Yes, they are permanently gone. Just download the novel again.

**Reply by hitguba248** [reply]
oof rip.  Guess I've gotta be more careful.  Thanks.
